docs-source/content/quickstart/install.md

```diff
@@ -37,40 +52,77 @@ $ helm install stable/traefik \
   --wait
 ```
 
+
+For helm version 3.x, you should have added the stable repository if it has not been done already.
+
+```bash
+$ helm repo add stable https://kubernetes-charts.storage.googleapis.com/
+```
+
+Create the namespace for traefik
+
+```bash
+$ helm create namesapce traefik
+```
+
+Install the traefik load balancer
+
+```bash
+$ helm install traefik-operator stable/traefik \
+    --namespace traefik \
+    --values kubernetes/samples/charts/traefik/values.yaml  \
+    --set "kubernetes.namespaces={traefik}" \
+    --wait
+```
+
 #### Install the operator.
 
 1.  Create a namespace for the operator:
 
-    ```bash
-    $ kubectl create namespace sample-weblogic-operator-ns
-    ```
+```bash
+$ kubectl create namespace sample-weblogic-operator-ns
+```
+
+2.	Create a service account for the operator in the operator's namespace:
 
-1.	Create a service account for the operator in the operator's namespace:
+```bash
+$ kubectl create serviceaccount -n sample-weblogic-operator-ns sample-weblogic-operator-sa
+```
 
-    ```bash
-    $ kubectl create serviceaccount -n sample-weblogic-operator-ns sample-weblogic-operator-sa
-    ```
+3.  Use `helm` to install and start the operator from the directory you just cloned:	 
 
-1.  Use `helm` to install and start the operator from the directory you just cloned:	 
+For helm 2.x
```

Comment:
'helm' needs to be capitalized, and you need a colon at the end of this line

Reply:
Fixed